default/stringtables/en.txt

```diff
@@ -11053,7 +11053,7 @@ PLC_STOCKPILE_LIQUIDATION
 Stockpile Liquidation
 
 PLC_STOCKPILE_LIQUIDATION_DESC
-Doubles stockpile use capacity added by stockpile-focused planets.
+Doubles planetary stockpile use capacity of stockpile-focused planets which increases the imperial stockpile use limit.
```

Comment:
This is more confusing after the change... It looks like "planetary stockpile" a thing...

Maybe `Doubles the contribution of stockpile-focused planets to the imperial stockpile use limit.`

Reply:
did some crosslinking, fixed up the map window description and added a new encyclopedia entry for imperial stockpile

stockpile meter and imperial stockpile article are now the nexi of stockpile information